v2/card_pools/startup.json

```diff
@@ -45,5 +40,18 @@
       "uprising",
       "uprising_booster_pack"
     ]
+  },
+  {
+    "id": "startup_borealis",
+    "format_id": "startup",
+    "name": "Startup (Borealis)",
+    "card_cycle_ids": ["borealis", "system_gateway", "system_update_2021"],
+    "card_set_ids": [
+      "midnight_sun",
+      "midnight_sun_booster_pack",
```

Comment:
Minor point, it's not actually important either way, but do we want to include the booster packs in set lists where those packs are strict subsets of other sets in the list? In this case the set of cards doesn't change if we remove MSBP from the list.

Reply:
right now we have a test that if a cycle is specified, all packs in the cycle are specified to make sure we don't miss things.  we can discuss revisiting that, but i don't wanna do that in this PR.